Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -41,7 +41,64 @@ var statusbar = {
         }
 }
 
+var copyProgressObjects = {};
+var copyObjectsManager = {
+    init: function() {
+        copyProgressObjects = {};
+    },
+    addCopyStats: function (uri, loadedData, totalData
+   )  { 
```

Comment:
I'm going to fix that in the next pull request, sometime in the next few days. I'm working on the manager for uploads. 


If everything else is ok, is it approvable in this condition?





On Fri, Oct 2, 2015 at 10:17 AM -0700, "suwatch01" <notifications@github.com> wrote:












In Kudu.Services.Web/Content/Scripts/FileBrowser.js:
> @@ -41,7 +41,64 @@ var statusbar = {
>          }
>  }
>  
> +var copyProgressObjects = {};
> +var copyObjectsManager = {
> +    init: function() {
> +        copyProgressObjects = {};
> +    },
> +    addCopyStats: function (uri, loadedData, totalData
> +   )  { 



yes.



—
Reply to this email directly or view it on GitHub.